front/app/components/SignUpIn/AuthProviders.tsx

```diff
@@ -137,6 +137,13 @@ const AuthProviders = memo<Props & WrappedComponentProps>(
     const phone =
       !isNilOrError(tenant) && tenant.attributes.settings.password_login?.phone;
 
+    const renderPasswordAuthButton =
```

Comment:
Is there a better name for this? Currently it sounds like this already renders the button.

Reply:
I think it's pretty good now, but we generally prefix statements that evaluate to a boolean with `is` so perhaps something like `isPasswordSignupAllowed`

Reply:
Good point, I amended that change.